src/Microsoft.AspNet.Hosting/HostingServices.cs

```diff
@@ -17,16 +17,19 @@ public static class HostingServices
         private static IServiceCollection Import(IServiceProvider fallbackProvider, Action<IServiceCollection> configureHostServices)
         {
             var services = new ServiceCollection();
-            if (configureHostServices != null)
-            {
-                configureHostServices(services);
-            }
             var manifest = fallbackProvider.GetRequiredService<IServiceManifest>();
             foreach (var service in manifest.Services)
             {
                 services.AddTransient(service, sp => fallbackProvider.GetService(service));
             }
+
```

Comment:
Can you also add a new test that verifies the additional services are found in the manifest? :)